### DISCUSSION TITLE
null 병합 연산자란 무엇인가요?

### DISCUSSION THREAD

**Discussion by pipisebastian** [publdaze/js-study#15] [타입변환과 단축 평가]
null 병합 연산자란 무엇인가요?
- 논리합 연산자(||)와 null 병합 연산자의 차이를 설명해주세요.

**Comment by publdaze** [comment]
> - null 병합 연산자란 무엇인가요?
- 좌항이 null이거나 undefined이면 우항을 반환하는 연산자
> - 논리합 연산자(||)와 null 병합 연산자의 차이를 설명해주세요.
- 논리합 연산자는 좌항이 falsy 값인 경우 우항 평가하고, null 병합 연산자는 좌항이 null이나 undefined인 경우 우항 평가함

**Comment by pipisebastian** [comment]
>null 병합 연산자란 무엇인가요?

좌항의 피연산자가 null/undefined 이라면 우항의 피연산자를 반환하고, 아닐 경우 좌항의 피연산자를 반환한다.

> 논리합 연산자(||)와 null 병합 연산자의 차이를 설명해주세요.

논리합 연산자(||)는 좌항이 Falsy값이면 우항의 피연산자를 반환하지만, null병합 연산자는 좌항이 undefined/null일 경우에만 우항의 피연산자를 반환합니다.
